README.md

```diff
@@ -3,15 +3,44 @@
 [![License: LGPL v3.0](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0)
 
 # Tokenbridge
-Monorepository for the token bridge.
+Welcome to the **POA Token Bridge** monorepository!
+
+Please note that this repository as a **work in progress**.
+
+## Overview
+
+The POA Token Bridge allows users to transfer assets between two chains in the Ethereum ecosystem. It is composed of several elements which are contained within this monorepository.
+
+For a complete picture of the POA Token Bridge functionality, it is useful to explore each subrepository.
 
 ## Structure
 
 Sub-repositories maintained within this monorepo are listed below.
 
 | Sub-repository | Description |
 | --- | --- |
-| [oracle](oracle/README.md) | Oracle responsible for listening to bridge related events and authorizing asset transfers |
+| [Oracle](oracle/README.md) | Oracle responsible for listening to bridge related events and authorizing asset transfers. |
+| [UI](bridge-ui/README.md) | DApp interface to transfer tokens and coins between chains. |
+| [Deployment](deployment-bridge/README.md) | Ansible playbooks for deploying cross-chain bridges. |
```

Comment:
I would recommend the change the order:
1. Oracle
2. UI
3. Monitor
4. Deployment

Reply:
Done.